recipe/meta.yaml

```diff
@@ -9,14 +9,15 @@ source:
   sha256: fae4cffc040921b8a2d60c6cf0b5d662c1190fe54d718271db4eb17d44a185b7
 
 build:
-  number: 1
+  number: 2
   noarch: python
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
:+1: I like the --ignore-installed. I'll give this a shot. Thanks for sticking with this one.

Reply:
If this works we (conda-forge) should start migrating to use it instead of `python setup.py install`.

Reply:
I have built a pytz package with this recipe and can confirm that my custom package can be built and then installed and run with this pytz package. I can confirm that the pytz package does not install an egg, instead installing a wheel. As far as I am concerned issue #14 is resolved.

Reply:
Thanks for the feedback @microe! For some odd reason CircleCi is taking too long but we'll merge this ASAP.